popup-portfolio.css

```diff
@@ -0,0 +1,285 @@
+.popuppt {
+  background-color: #c1c7d0;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 9;
+  overflow: auto;
+  padding: 16px;
+  opacity: 1;
+}
```

Comment:
- [ ] Kindly style you desktop view of the pop-up to fit into the white background s in the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/217628069-2ba48f92-b627-4d9b-bbb6-e238fb1c0473.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/217628499-906f9b4b-e65a-44da-9b0d-782832c82e9c.png)

Reply:
@Munsa1  could you help me with which width screen is that one? It worked well in 1080.